src/server/utilities/cspHeader/index.js

```diff
@@ -245,7 +245,7 @@ const directives = {
       'https://www.instagram.com', // Social Embeds
```

Comment:
is it possible that we might get the same issue with instagram?

Reply:
Should be ok, we encountered a problem with Instagram on Safari awhile ago, but it was fixed.